User_id/css/global.css

```diff
@@ -0,0 +1,23 @@
+html,
+body {
+	height: 100%;
+}
+
+.wrapper {
+	display: flex;
```

Comment:
Indent is different
https://github.com/AlexeyDunnikov/JS_Lessons/pull/4/files#diff-f410e6f3aca010bdb00f143ab579ece1d370329514bfa1a32bd49b44095b8ff1R19